### PR TITLE
Rev bounds

### PR DESCRIPTION
**Pull request by archmoj**
This PR is another module required to be updated to enable drawing the scenes with reversed bounds. In plotly.is such ranges could be set using autorange:'reversed' or range:[greater value, smaller value] .
Please refer to plotly [issue](https://github.com/plotly/plotly.js/issues/1940) and this [PR](https://github.com/plotly/plotly.js/pull/3071).
@alexcjohnson

### CODE REVIEW

File: shaders/fragment.glsl [hunk 1]
```diff
@@ -16,10 +17,9 @@ varying vec3 lightDirection, eyeDirection, surfaceNormal;
 varying vec4 vColor;
 
 void main() {
-  if (kill > 0.0 ||
-    any(lessThan(worldCoordinate, clipBounds[0])) || any(greaterThan(worldCoordinate, clipBounds[1]))) {
-    discard;
-  }
+  if (kill > 0.0) discard;
+
+  if (outOfRange(clipBounds[0], clipBounds[1], worldCoordinate)) discard;
```
Comment (alexcjohnson) [inline shaders/fragment.glsl]: This is definitely more readable... but it does seem to be established practice to minimize `if` statements:

> In particular, if you can remove an if statement from a shader, that will make it run faster.

https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/WebGL_best_practices

So probably better to collapse it back to a single `if`.

**Comment by archmoj** [comment]
Thank you for the observation @alexcjohnson 
The if statement for this module is revised.

**Comment by alexcjohnson** [comment]
heh https://github.com/gl-vis/gl-surface3d/pull/12/commits/a268ac00f290ee46a00851e2faebab47097e2563 got one of the double-ifs, but there's another (the one I actually commented on 😈 )

**Comment by alexcjohnson** [comment]
Thanks, now you've got it! 💃

**Pull request merged**